packages/web-components/fast-foundation/src/design-system/design-system.ts

```diff
@@ -3,6 +3,7 @@ import {
     FASTElementDefinition,
     PartialFASTElementDefinition,
 } from "@microsoft/fast-element";
+import { FoundationElement } from "..";
```

Comment:
I *think* we'll want a `/` here - it may be fine but I know we've seen issues related to pathing without one previously.

```suggestion
import { FoundationElement } from "../";
```

Reply:
VSCode. I'll fix shortly.

Reply:
This revealed some circular module issues. Working to fix that now.